<?xml version="1.0"?>
<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node">
    <param name="video_device" value="/dev/video0"/>
    <param name="image_width" value="640"/>
    <param name="image_height" value="480"/>
    <param name="pixel_format" value="yuyv"/>
    <param name="framerate" value="30"/>
  </node>
</launch>
